test/functional/cypress/specs/companies/activity-feed-spec.js

```diff
@@ -199,6 +199,14 @@ describe('Company activity feed', () => {
         )
     })
 
+    it('should have the Analytics id', () => {
+      cy.get('#ga-company-details-matching-prompt').should(
+        'have.attr',
+        'id',
+        'ga-company-details-matching-prompt'
+      )
+    })
```

Comment:
No need to verify ID as we got the element using the ID, we could just use `exist` here:
```suggestion
      cy.get('#ga-company-details-matching-prompt').should('exist')
```